python/unityagents/environment.py

```diff
@@ -95,11 +96,21 @@ def __init__(self, file_name, worker_id=0,
         else:
             logger.debug("This is the launch string {}".format(launch_string))
             # Launch Unity environment
-            proc1 = subprocess.Popen(
-                [launch_string,
-                 '--port', str(self.port),
-                 '--seed', str(seed)])
-
+            if docker_training == False:
+                proc1 = subprocess.Popen(
+                    [launch_string,
+                     '--port', str(self.port),
+                     '--seed', str(seed)])
+            else:
+                docker_ls = ("exec xvfb-run --auto-servernum"
```

Comment:
A comment explaining docker_ls would be very helpful here, particularly with regards to the dimensions (640x480x24).